includes/elements/container.php

```diff
@@ -128,6 +130,16 @@ protected function add_render_attributes() {
 				$is_nested_class_name,
 			],
 		] );
+
+		if ( $this->get_data( 'isInner' ) ) {
+			return;
+		}
+
+		// Todo: remove in future version.
+		// Remove together with support for physical properties inside the Mega Menu & Nested Carousel widgets.
+		$this->add_render_attribute( '_wrapper', [
+			'data-core-v316-plus' => 'true',
```

Comment:
I have added an attribute on the parent container so that we can more easily distinguish between the old and the new version. I need that distinction inside the Nested Widgets in pro.